Data/Sys/GameSettings/SBLE5G.ini

```diff
@@ -0,0 +1,8 @@
+# SBLE5G - A Boy and His Blob
+
+[OnFrame]
+$Fix Bean Menu
```

Comment:
Maybe this patch name should mention it's an input issue. Without more context or investigation I think it would be hard to figure out what this patch is even fixing. They might think it's a graphical issue or something.